go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/codeready-toolchain/toolchain-operator
 require (
 	github.com/codeready-toolchain/api v0.0.0-20191009081803-8f4e395353de
 	github.com/codeready-toolchain/toolchain-common v0.0.0-20191010043304-822e291d04cb
+	github.com/docker/docker v1.13.1
```

Comment:
I guess this pkg may not be needed, try running `go mod tidy` once and see if that helps.

Reply:
ah right, that's probably a left over after a wrong import :/

Reply:
fixed in a9569a6